lib/api/client-commands/pause.js

```diff
@@ -42,7 +42,7 @@ Pause.prototype.command = function(ms, cb) {
   // If we don't pass the milliseconds, the client will
   // be suspended indefinitely, until the user presses some
   // key in the terminal to resume it.
-  if (!ms) {
+  if (!ms && ms !== 0) {
```

Comment:
Should we make it `if (ms === undefined) {` instead? Because that's only when we should pause indefinitely (if the first argument is `undefined`). If something else is passed as the first argument which is falsy (`null`, `false` for ex.), then we should just let it go to the `else` clause and let `setTimeout` through an error, instead of pausing indefinitely?

Reply:
Yes, Will make changes.